include/llbuild/BuildSystem/BuildValue.h

```diff
@@ -324,7 +348,7 @@ class BuildValue {
   bool isTarget() const { return kind == Kind::Target; }
 
   std::vector<StringRef> getDirectoryContents() const {
```

Comment:
We should give this a different name for stale file removal, like `getStaleFileList()`